docs/sources/developers/http_api/alerting.md

```diff
@@ -27,7 +27,7 @@ This topic is relevant for the [legacy dashboard alerts](/docs/grafana/v8.5/aler
 
 If you are using Grafana Alerting, refer to [Alerting provisioning API]({{< relref "./alerting_provisioning" >}})
 
-You can find Grafana Alerting API specification details [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json). Also, refer to [Grafana Alerting alerts documentation]({{< relref "../../alerting" >}}) for details on how to create and manage new alerts.
+You can find Grafana Alerting API specification details [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json). Also, refer to [Grafana Alerting alerts documentation][] for details on how to create and manage new alerts.
```

Comment:
Is there supposed to be a link for alerting doc?

Reply:
The link comes later in https://github.com/grafana/grafana/pull/75875/files#diff-86f720a19a48ae01de27f4780059be5883b7d962422e408d84dc78ba42edd260R182-R183